lib/qa/authorities/linked_data/search_query.rb

```diff
@@ -35,7 +35,7 @@ def search(query, language: nil, replacements: {}, subauth: nil, context: false)
         raise Qa::InvalidLinkedDataAuthority, "Unable to initialize linked data search sub-authority #{subauth}" unless subauth.nil? || subauthority?(subauth)
         @context = context
         @language = language_service.preferred_language(user_language: language, authority_language: search_config.language)
-        url = authority_service.build_url(action_config: search_config, action: :search, action_request: query, substitutions: replacements, subauthority: subauth)
+        url = authority_service.build_url(action_config: search_config, action: :search, action_request: query, substitutions: replacements, subauthority: subauth, language: @language)
```

Comment:
Same here.  Needs to pass `@language` which is the preferred langauge, instead of the user/http language which may be nil if not set.